tsconfig.json

```diff
@@ -24,13 +24,12 @@
     "jsx": "react",
     "paths": {
       "gosling.js": ["./src/index.ts"],
-      "@gosling.schema": ["./src/core/gosling.schema.ts"],
-      "@higlass.schema": ["./src/core/higlass.schema.ts"],
-      "@gosling-mouse-event": ["./src/gosling-mouse-event/index.ts"],
-      "@gosling-track": ["./src/gosling-track/index.ts"],
-      "@gosling-genomic-axis": ["./src/gosling-genomic-axis/index.ts"],
-      "@gosling-brush": ["./src/gosling-brush/index.ts"],
-      "@gosling-lang/dummy-track": ["./src/dummy-track/index.ts"],
+      "@gosling-lang/gosling-schema": ["src/gosling-schema/index.ts"],
+      "@gosling-lang/higlass-schema": ["src/higlass-schema/index.ts"],
+      "@gosling-lang/gosling-track": ["./src/tracks/gosling-track/index.ts"],
+      "@gosling-lang/gosling-genomic-axis": ["./src/tracks/gosling-genomic-axis/index.ts"],
+      "@gosling-lang/gosling-brush": ["./src/tracks/gosling-brush/index.ts"],
+      "@gosling-lang/dummy-track": ["./src/tracks/dummy-track/index.ts"],
```

Comment:
We could also add a subpackage that combines these tracks, e.g., `@gosling-lang/tracks`.

Similarly, we can have separate packages for each of the data fetchers, e.g., `@gosling-lang/vcf-data-fetcher`.

These can be a follow-up PR